queue_job/models/queue_job.py

```diff
@@ -657,9 +610,13 @@ def _parse_retry_pattern(self):
             retry_pattern = {}
         return retry_pattern
 
+    @tools.ormcache("name")
+    def get_by_name(self, name):
+        return self.search([("name", "=", name)], limit=1)
```

Comment:
this is caching the full object. IMO you should cache the ID or a dict w/ full info to be used in `job_config`

Reply:
thaaaanks!

Reply:
Added the id in JobConfig, can you check the fixup so I squash?